ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayModal.tsx

```diff
@@ -276,11 +280,13 @@ export default function NewReplayModal({ functionSlug, isOpen, onClose }: NewRep
               <span className="font-medium text-slate-800">
                 {isLoading ? (
                   <Placeholder className="top-px inline-flex h-4 w-4 bg-slate-200" />
-                ) : (
+                ) : isTimeRangeValid ? (
                   selectedRunsCount.toLocaleString(undefined, {
                     notation: 'compact',
                     compactDisplay: 'short',
                   })
+                ) : (
+                  '-'
```

Comment:
From @anafilipadealmeida's feedback, can we make this blank? Maybe using a nbsp to prevent layout shift?